pushapkscript/jarsigner.py

```diff
@@ -48,4 +47,7 @@ def _pluck_configuration(context):
         'dep': 'dep',
     })
     android_product = extract_android_product_from_scopes(context)
-    return binary_path, keystore_path, certificate_aliases, android_product
+    certificate_alias = context.task['payload'].get(
```

Comment:
Nit: Let's either fail or warn if both are defined. I foresee our future selves being confused about what alias it takes. Having something in the logs would help us.

Reply:
Oooh good point!